js/private/node-patches/src/fs.ts

```diff
@@ -634,23 +634,7 @@ export const patcher = (fs: any = _fs, roots: string[]) => {
             }
             if (isEscape(loc, next)) {
                 // this hop takes us out of the guard
-                return nextHop(next, (next2: string | false) => {
```

Comment:
If I recall correct this was one of the major spots wasting time.

Reply:
Sounds about right